queuebot/cogs/queue/__init__.py

```diff
@@ -292,11 +289,11 @@ def has_emoji_slots(self, guild: discord.Guild) -> bool:
                         ))
 
             if self.config.verbose_vs:
-                mapped_lines = [NUMERIC_EMOTES[temp_emotes.index(e)] + str(e) for e in temp_emotes]
-                emote_sequence = VERBOSE_VS_JOINER.join(mapped_lines)
+                emote_sequence = VERBOSE_VS_JOINER.join(
+                    [str(i) + "\n{COMBINING ENCLOSING KEYCAP}" + str(e) for i, e in enumerate(temp_emotes, 1)]
```

Comment:
Since this project is locked to Python 3.6 anyway, an f-string could be used here.